src/lib.rs

```diff
@@ -58,6 +59,9 @@ pub struct WasiVirt {
     pub random: Option<bool>,
     /// Disable wasm-opt run if desired
     pub wasm_opt: Option<bool>,
+    /// Path to compose component
+    pub compose: Option<String>,
```

Comment:
Perhaps when setting this option, `virt.finish()` should return the composition instead of the virt component?

Or alternatively we could support a `virt.filter(component)` which can do the disabling of unused subsystems, and a `virt.compose()` which can then do the composition without needing the CLI?

Reply:
`.finish()` now performs the composition, and moved the subsystem filtering into a separate `.filter_imports()`. Updated the readme as well.